emission/core/wrapper/common_place.py

```diff
@@ -0,0 +1,16 @@
+import logging
+import emission.core.wrapper.wrapperbase as ecwb
+
+class CommonPlace(ecwb.WrapperBase):
+    props = {"coords" : ecwb.WrapperBase.Access.WORM, # geojson coordinates that represent the place
+             "successors" : ecwb.WrapperBase.Access.WORM, # set of CommonPlaces connected to this location
+             "user_id" : ecwb.WrapperBase.Access.WORM,
+             "common_place_id" : ecwb.WrapperBase.Access.WORM
```

Comment:
I use it in line 73 of common_trip_queries.py when I'm looking for successors 
        _id = esdctp.make__id(user_id, place_id, suc)
        trip = esdctp.get_common_trip(_id)

but I can just use a mongo query, I don't need a special _id